dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+              content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Restaurant Page</title>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
+        <script src="menu.bundle.js"></script>
+        <script src="contact.bundle.js"></script>

```

Comment:
- [ ] Kindly delete these script tags. There should only be one bundled js file

Reply:
Hi, thanks for the review. You should really look into the webpack documentation where it states that if you have several JS files that are being used you need to place them as points of entry and then bundle them in main.js

https://webpack.js.org/guides/output-management/

This will help you not to make wrong reviews, i will make the changes and when the project it's approved i will revert them back,

Thanks

Reply:
Done